.emacs.d/settings/custom.el

```diff
@@ -8,7 +8,7 @@
 
 ; color theme
 (add-to-list 'custom-theme-load-path (make-plugin-path "color-theme-solarized"))
-(load-theme 'solarized-dark 1)
+(load-theme 'solarized 1)
 (setq solarized-termcolors 256)
```

Comment:
Won't you need another line that will set `'dark` or `'light` theme?

Something along this line: `(set-frame-parameter nil 'background-mode 'dark)`?